pkg/apis/kubedirector/v1beta1/kubedirectorcluster_types.go

```diff
@@ -155,6 +156,14 @@ type ClusterStorage struct {
 	StorageClass *string `json:"storageClassName,omitempty"`
 }
 
+// BlockStorage defines the block storage type, path, and optionally size, if any, to be used
+// for mounting a block volume in a role.
+type BlockStorage struct {
+	StorageClass *string `json:"storageClassName,omitempty"`
+	Path         *string `json:"path,omitempty"`
```

Comment:
nit: let's keep this consistent with the property name in the status which is blockdevicepath